persistence/sql.go

```diff
@@ -40,12 +39,111 @@ func connectAndInitializeDatabase(postgresUrl string, schema string) error {
 	if _, err = db.Exec(ctx, fmt.Sprintf("%s %s", SetSearchPathTo, schema)); err != nil {
 		return err
 	}
+	// pgx.MigrateUp(options, "persistence/schema/migrations")
 	if _, err = db.Exec(ctx, fmt.Sprintf(`%s %s %s`, CreateTableIfNotExists, TableName, TableSchema)); err != nil {
 		log.Fatalf("Unable to create %s table: %v\n", TableName, err)
 	}
+	if err := InitializeTables(ctx, db); err != nil {
+		log.Fatal(err.Error())
+	}
+	return nil
+}
 
-	// TODO(olshansky;github.com/pokt-network/pocket/issues/77): Enable proper up and down migrations
-	// pgx.MigrateUp(options, "persistence/schema/migrations")
+func InitializeTables(ctx context.Context, db *pgx.Conn) error {
```

Comment:
Just FYI: once I implement migrations, we'll probably be able to delete all of this.

Reply:
Resolving because I added a TODO inline.